src/cmd/ksh93/tests/pty.sh

```diff
@@ -48,7 +48,7 @@ x=$( $SHELL <<- \EOF
         #sleep 1
         jobs
         kill $$
-    EOF
+	EOF
```

Comment:
FYI, A better fix would have been to leave the spaces and change the `<<-` to `<<#`. Now this file has a single invisible tab character.

Reply:
What is the policy on spaces vs tabs for the shell scripts ?
It is a mix of both everywhere.

Reply:
I left tabs in `<<-` sections for readability. I was not aware about the ` <<# ` syntax. I think we can replace all occurences of `<<-` with `<<#` and replace all tabs with spaces. The only exception is the output of `typeset` command.

Reply:
Tabs cause more problems than they're worth. And mixing tabs and spaces is especially bad. We haven't established a policy for scripts. But the policy for the C source files is no tabs. I'd prefer that be the policy for the scripts as well.

Reply:
I switched from tabs to spaces 30 years ago and never looked back.
I am glad the consensus is to prefer spaces over tabs in this project.